test/linalg/tridiag.jl

```diff
@@ -72,7 +75,7 @@ for elty in (Float32, Float64, Complex64, Complex128, Int)
         Ts = SymTridiagonal(d, dl)
         Fs = full(Ts)
         invFsv = Fs\v
-        Tldlt = ldltfact(Ts)
+        Tldlt = factorize(Ts)
```

Comment:
for reference: https://github.com/JuliaLang/julia/blob/181c19f5de971594fcd2b79dbf0789d159b579c2/base/linalg/ldlt.jl#L33

PR LGTM